src/imp/noop_imp.js

```diff
@@ -1,18 +1,25 @@
+function NoopSpanContextImp() {
```

Comment:
@yurishkuro my understanding (@bcronin can correct me here) is that the `imp` interface is not quite the same as the user-facing interface in that some params are normalized, etc. (e.g., the startSpan() `fields` param)